src/browser/modules/Sidebar/GuidePicker.tsx

```diff
@@ -54,13 +54,12 @@ const GuidePicker = ({
     </DrawerBrowserCommand>
     in the code editor.
     <MarginTop pixels={25}>
-      <DrawerSubHeader as="div" /* prevents guide styling of h5*/>
```

Comment:
The comment went missing all together now, I think it is nice to have that since I am thinking about why this as=div is set here otherwise.

Reply:
Sorry for misunderstanding the previous comment.

Reply:
No problem, now it looks good 👍